Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
```

Comment:
@rhcarvalho yeah that's what I'd have expected (and I'm sure I've seen and done that before). When I tried it though, it didn't seem to work. I'm sure I probably had some typo though, because even the comments that I put in were throwing errors, saying command not found.